### ISSUE TITLE
year tag gets doubled

### ISSUE THREAD

**Issue by cubetronic** [audacity/audacity#401]
id3v3.2 or whatever, for example...  
i edit a song and export it, it already has tag data, i click ok.  
but what was date `2010-01-01` is now `2010-01-01; 2010-01-01`  
audacity added an extra `; 2010-01-01`
so now when i try to view the year on some mp3 players, like vanilla music player, it won't show the year. (and to make matters worse, flushing the cache in vanilla music player is not working either, so even though i've fixed the file, it still shows the old metadata)

used latest release of audacity downloaded like a few days ago

**Comment by JamesCrook** [comment]
Are you on mac or windows?
Perhaps you could send us a small example, and tell us the exact steps to make the problem happen.  Thanks.

**Comment by petersampsonaudacity** [comment]
I could not reproduce this on W10

**Comment by cubetronic** [comment]
I just reproduced it. 
Audacity 2.3.3 
Windows 10 Pro Version 10.0.18363 Build 18363 (Nov 2019) 
id3v2.3  
  
1. Start with an mp3 file tagged with id3v2.3, with a year like "1998" for example.
2. Drag that file into Audacity.
3. Edit the file however you wish. For me, I like to split it and remove audio I don't want.
4. Highlight the audio you want, like with Ctrl-A for everything.
5. Click on File | Export > Export as mp3
6. I just went with the quality presets, renamed the file, selected a folder, and clicked Save.
7. Current mp3 tag info will be displayed (note year "1998"). I just click OK.
8. Now open that file in a DIFFERENT MUSIC PLAYER, like Foobar2000 (Windows 10), or Vanilla Music (Android) and you will see "1998; 1998" in the Year tag. The duplicate year will NOT be shown if you drag the new file into Audacity and view Metadata... (technically this is *another bug*, because Audacity should be showing what is truly there).

Foobar2000 renders the year as "1998, 1998" in the player but "1998; 1998" in the Properties Editor. This may be similar to how Audacity is only showing one year when the actual tag data is "1998; 1998".

**Comment by JamesCrook** [comment]
Thanks.  We'll look into it.

**Comment by petersampsonaudacity** [comment]
As before I cannot reproduce this (using these latest Steps to reproduce) testing on W10 with 2.3.3.

My music player of choice (iTunes) as Step 8 shows only a single "1998"

When I examine the MP3 file's properties with Windows Explored it too shows only a single "1998"

**Comment by petersampsonaudacity** [comment]
As before I cannot reproduce this (using these latest Steps to reproduce)
testing on W10 with 2.3.3.

My music player of choice (iTunes) as Step 8 shows only a single "1998"

When I examine the MP3 file's properties with Windows Explored it too shows
only a single "1998"

Peter.

**Comment by cubetronic** [comment]
Windows Explorer is not showing you the actual content of the data - but instead giving you a nicely formatted version of that data. I believe iTunes is doing the same thing, but I don't use iTunes.

Like I said, this is noticeable in the 3rd Party Programs I use:
- Foobar2000
- Vanilla Music Player

For good measure, I just went ahead and downloaded an mp3 tag program:
- *Mp3tag*  
This confirms the bug, **and** the hard-to-detect nature of the bug
If you drag your newly-created mp3 file into Mp3tag...
`Mp3tag version:
[2019-10-19]  REL: VERSION 2.99a (for Windows 10/8.1/8/7/2008/Vista/2003/XP SP3)`
Then it will show `1998\\1998` on the left panel, but if you right-click on the file and go to Properties, it will just show `1998`. WHEREAS, if you use an un-altered file, it will show just `1998` in both places.
Of course this is a bit different, with two backslashes. I honestly don't know what's *actually* in that data field. But I do know the data is mangled after exporting from Audacity.

Also, some programs call it "date", some "year"... whatever

**Comment by petersampsonaudacity** [comment]
So I installed MP3tag to investigate this further

And yes at step 8 with MP3tag (invoked from Windows Explorer
The Year field indeed shows 1998\\1998

But the so does the MP3 file that I created at Step 5 (the original MP3 I
created)
show as 1998\\1998

So it appears your Step 8 is not adding an extra "1998" as far as MP3tag is
concerned.

So I remain puzzled .

-----------------------------------------------

BTW - we know that Audacity is a poor manager of metadata, particularly of
pr-existing
metadata when an MP3 is imported - for example the artwork is discarded.

I have a QA page on the Audacity Wiki (in addition to our Bigtracker) to
keep an eye on
metadata issues in Audacity:
https://wiki.audacityteam.org/wiki/User:PeterSampson/Metadata_issues

Peter.
